internal/k8s/kctl/deploy.go

```diff
@@ -19,12 +21,23 @@ type Wait interface {
 	Process(labels []string)
 }
 
+func pause() {
+	time.Sleep(2 * time.Second)
+}
+
 // Apply adds/updates the plugin in a k3s/k8s cluster
 func Apply(config Config, plugin plugins.PluginSpec, evt Wait) error {
-	err := handleYaml(config, apply, plugin)
-	if err != nil {
-		return err
+	_ = createNameSpace(config, plugin.NameSpace)
+	pause()
+	for _, yamlSpec := range plugin.Yaml {
+		err := execute(config, k3sExec, kubectl, apply,
+			decodeType(yamlSpec.Type), yamlSpec.URL, "-n", plugin.NameSpace)
+		if err != nil {
+			log.Fatalf("Error during create: %s", err.Error())
```

Comment:
You can return this error to the caller

Reply:
I voluntary want that the cycle continues. There could be temporary problems ( especially for the kubeflow installation) there are a lot of retries. So I'd leave it as it then we will see.

Reply:
Fatalf panics with error. You might want to println in that case